src/provider/ChannelModel.ts

```diff
@@ -1,80 +1,79 @@
 import {Identity} from 'openfin/_v2/main';
```

Comment:
Do we want to be using `Transport` objects at this level? They feel more like things to use as close to the point we're sending stuff across the wire as possible

Reply:
A little bit of a trade-off here. Doing the transport conversion lower-down in the chain may require exposing a bit more of the internals of the model - things that could otherwise be encapsulated.

Need to decide if we want this class being the `APIHandler` endpoint for those calls, or if we want something more at the `index`/`APIHandler`-level to do the conversion.

The implementation story would be a good time to decide this, as I imagine a lot of this will be changing anyway.